### PR TITLE
chore(My Portal): [#174578457] Adds a feature flag for myportal integration

### PR DESCRIPTION
**Pull request by CrisTofani**
## Short description
This PR adds the MyPortal Integration feature flag on env files

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -36,6 +36,7 @@ INSTABUG_TOKEN='5c2d0f12fa12f9afc535585e5b7a9e79'
 MIXPANEL_TOKEN='0cb505dace6f4b3ceb9e17c7fcd7c66f'
 GCM_SENDER_ID='260468725946'
 BONUS_VACANZE_ENABLED=NO
+MYPORTAL_INTEGRATION_ENABLED=NO
```
Comment (Undermaken) [inline .env.example:39]: could it be like this for shortness?
```suggestion
MYPORTAL_ENABLED=NO
```

File: ts/config.ts [hunk 2]
```diff
@@ -45,6 +45,9 @@ export const instabugToken: string = Config.INSTABUG_TOKEN;
 export const bonusVacanzeEnabled: boolean =
   Config.BONUS_VACANZE_ENABLED === "YES";
 
+export const myPortalIntegrationEnabled: boolean =
```
Comment (Undermaken) [inline ts/config.ts:48]: some of above
```suggestion
export const myEnabled: boolean =
```

**Review by Undermaken — CHANGES_REQUESTED** [review]

File: .env.example (hunk 1 above)
Comment (CrisTofani) [reply .env.example:39]: Yes sounds good!

File: ts/config.ts (hunk 2 above)
Comment (CrisTofani) [reply ts/config.ts]: b3803a4 renames it!

File: .env.example (hunk 1 above)
Comment (CrisTofani) [reply .env.example]: b3803a4 renames it!

**Review by Undermaken — APPROVED** [review]

**Pull request merged**